lib/packagekit-glib2/pk-enum.h

```diff
@@ -563,6 +564,7 @@ typedef enum {
 	PK_GROUP_ENUM_ACCESSORIES,
 	PK_GROUP_ENUM_ADMIN_TOOLS,
 	PK_GROUP_ENUM_COMMUNICATION,
+	PK_GROUP_ENUM_DESKTOP_DDE,
```

Comment:
Please move that below `PK_GROUP_ENUM_NEWEST`, otherwise this will break ABI!

Reply:
Oh sorry I completely forget about this is also a library so ABI is a thing. The PR is here: https://github.com/PackageKit/PackageKit/pull/629. And I think maybe we should put it above NEWEST? (let's discuss this in that PR)